docs/usage/configuration-options.md

```diff
@@ -698,6 +698,18 @@ You can configure this to `true` if you prefer Renovate to close an existing Dep
 The labels only get updated when the Dependency Dashboard issue updates its content and/or title.
 It is pointless to edit the labels, as Renovate bot restores the labels on each run.
 
+## dependencyDashboardOSVVulnerabilitySummary
```

Comment:
Shouldn't we mention that it requires the osv vulnerabilities feature to be turned on, or this will do nothing?

Reply:
The options are independent.    
The user can use both as well as one or the other.

Reply:
Let's mention that then?